tools/python/model_validation/validation_config.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
we are building a general tool. This model specific config should not appear in the folder

Reply:
updated the configs to include only gen-ai configs + some defaults to facilitate future use cases.